### DISCUSSION TITLE
Adding multiple webhooks

### DISCUSSION THREAD

**Discussion by HaykoAnderson** [DisboxApp/web#138] [Ideas]
Could you try to make so we can put multiple webhooks (for the same channel) so the upload is faster and maybe the download speed could be faster too ?

**Comment by abusivePenguin** [comment]
maybe this could be implemented by splitting the workload between webhooks? so for a 1 GB file with a channel with 2 webhooks, it each webhook will only need to upload 500MB. Would this be possible?
